tests/lib/MVC/Symfony/EventListener/CustomMatcher.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Core\MVC\Symfony\EventListener;
+
+use Ibexa\Bundle\Core\SiteAccess\Matcher;
+use Ibexa\Core\MVC\Symfony\SiteAccess\Matcher\Map\URI;
+
+class CustomMatcher extends URI implements Matcher
```

Comment:
A different name might better indicate that this is used only in tests, and not an actual `CustomMatcher`.
```suggestion
class TestMatcher extends URI implements Matcher
```